src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -66,7 +66,7 @@
 
 		<!-- Additional message info-->
 		<div v-if="!isDeletedMessage" class="message-main__info">
-			<span v-if="!hideDate" class="date" :title="messageDate">{{ messageTime }}</span>
+			<span :class="['date', { 'date--hidden': hideDate }]" :title="messageDate">{{ messageTime }}</span>
```

Comment:
```suggestion
			<span :class="['date', { 'hidden-visually': hideDate }]" :title="messageDate">{{ messageTime }}</span>
```

Reply:
What would kick element out of render flow. I keep it like this specifically to reserve a certain amount of space for the date